e2e/testcafe-devextreme/tests/chat/messageList.ts

```diff
@@ -145,3 +146,45 @@ test('Messagelist should scrolled to the latest messages after being rendered in
     }],
   });
 });
+
+test('Messagelist with date headers', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'Messagelist with date headers.png', { element: '#container' });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  const userFirst = createUser(1, 'First');
+  const userSecond = createUser(2, 'Second');
+
+  const items = [{
+    timestamp: new Date('05.01.2024'),
```

Comment:
You declared '05.01.2024', but in the screenshot, I see the time as 24:00, which means that the date is at least '06.01.2024'. Could you please check this and create a bug card if needed? Also, it's strange to see the time in 24:00 format.

Reply:
I don't know why screenshots on gh have `24:00` time, localy it's 00:00. Probably gh have some unusual region or smth. 
Anyway in test we set date `05.01.2024` and day header has text `05.01.2024` => it's doesn't seem as a problem with day headers, probably we can create some card to research if there is an issue with time, but more likely it's just a region speccific stuff

Reply:
"probably we can create some card to research if there is an issue with time" - Let's do it

Reply:
a new card is added in toDiscuss